README.md

```diff
@@ -46,32 +46,31 @@ Further, in `samutev.conf` you can customize [cloudinit](https://cloudinit.readt
 ```
 ./samutev.sh -h
 Usage:
-    ./samutev.sh -h          # Display this help message.
-    ./samutev.sh -n <VM>     # new <VM> with masterless minion
-    ./samutev.sh -s <VM>     # new <VM> with minion and salt master. FIRST vm=saltmaster. minimum 2 vm's
-    ./samutev.sh -d <VM>     # delete <VM>
-    ./samutev.sh -l          # list   VM's
+        ./samutev.sh -h                  Display this help message
+        ./samutev.sh [-b] -n <VM>        new     <VM> with masterless minion
+                                                 (default: latest lts, -b => bionic)
```

Comment:
to use a letter for the release wil lead to problems.
- bionic somehow could be breezy too
- -h could be hoary or the one after groovy too
i suggest to use neutral "-r" as "release"

Reply:
I agree. I was thinking along the same lines and decided to go for an iterative approach and let the future-me deal with it. ;-)

So you suggest something like that: -r bionic? That's cool. I will update my PR accordingly.

Reply:
yes :-)

Reply:
I added a commit to address this. Now you can also use xenial (I limited it to LTS releases).

I tested all combinations:
20.04:
OKAY time ./samutev.sh -s "project-master:c4:m4:d3 project-app"
OKAY time ./samutev.sh -r 20.04 -s "project-master:c4:m4:d3 project-app"
OKAY time ./samutev.sh -r lts -s "project-master:c4:m4:d3 project-app"
OKAY time ./samutev.sh -r focal -s "project-master:c4:m4:d3 project-app"

18.04:
OKAY time ./samutev.sh -r 18.04 -s "project-master:c4:m4:d3 project-app"
OKAY time ./samutev.sh -r bionic -s "project-master:c4:m4:d3 project-app"

16.04:
OKAY time ./samutev.sh -r 16.04 -s "project-master:c4:m4:d3 project-app"
OKAY time ./samutev.sh -r xenial -s "project-master:c4:m4:d3 project-app"